ApplicationUser.cs

```diff
@@ -0,0 +1,7 @@
+public class HolyShiftUser
```

Comment:
HolyShiftUser should belong to the next PR not this one. Remove it from this PR.